frontend/app/module/input/PollingStation/PollingStation.test.tsx

```diff
@@ -0,0 +1,325 @@
+}
+
+const startPollingStationInput = async () => {
+  await router.navigate("/1/input/1");
+  expect(router.state.location.pathname).toEqual("/1/input/1");
+};
+
+const expectRecountedForm = async () => {
+  await waitFor(() => {
+    expect(screen.getByTestId("recounted_form")).toBeInTheDocument();
+  });
+};
+
+const fillRecountedForm = async () => {
+  await user.click(screen.getByLabelText("Nee, er was geen hertelling"));
+};
+
+const expectVotersAndVotesForm = async () => {
+  await waitFor(() => {
+    expect(screen.getByTestId("voters_and_votes_form")).toBeInTheDocument();
+  });
+};
+
+const fillVotersAndVotesForm = async (values?: Record<string, number>) => {
+  const total_votes = electionMockData.political_groups.length * 10;
+
+  const userValues = values ?? {
+    poll_card_count: total_votes,
+    proxy_certificate_count: 0,
+    voter_card_count: 0,
+    total_admitted_voters_count: total_votes,
+    votes_candidates_counts: total_votes,
+    blank_votes_count: 0,
+    invalid_votes_count: 0,
+    total_votes_cast_count: total_votes,
+  };
+
+  await userTypeInputs(user, userValues);
+};
+
+const expectDifferencesForm = async () => {
+  await waitFor(() => {
+    expect(screen.getByTestId("differences_form")).toBeInTheDocument();
+  });
+};
+
+const fillDifferencesForm = async () => {
+  await userTypeInputs(user, {
+    more_ballots_count: 0,
+    fewer_ballots_count: 0,
+    unreturned_ballots_count: 0,
+    too_few_ballots_handed_out_count: 0,
+    too_many_ballots_handed_out_count: 0,
+    other_explanation_count: 0,
+    no_explanation_count: 0,
+  });
+};
+
+const expectPoliticalGroupCandidatesForm = async (pgNumber: number) => {
+  await waitFor(() => {
+    expect(screen.getByTestId(`candidates_form_${pgNumber}`)).toBeInTheDocument();
+  });
+};
+
+const fillPoliticalGroupCandidatesVotesForm = async () => {
+  await userTypeInputs(user, {
+    "candidate_votes[0].votes": 5,
+    "candidate_votes[1].votes": 5,
+    total: 10,
+  });
+};
+
+const expectCheckAndSavePage = async () => {
+  await waitFor(() => {
+    expect(router.state.location.pathname).toEqual("/1/input/1/save");
+  });
+};
+
+const expectFeedbackError = async (code?: string) => {
+  await waitFor(() => {
+    expect(screen.getByTestId("feedback-error")).toBeInTheDocument();
+  });
+  if (code) {
+    expect(screen.getByText(code)).toBeInTheDocument();
+  }
+};
+
+const expectFeedbackWarning = async (code?: string) => {
+  await waitFor(() => {
+    expect(screen.getByTestId("feedback-warning")).toBeInTheDocument();
+  });
+  if (code) {
+    expect(screen.getByText(code)).toBeInTheDocument();
+  }
+};
+
+const expectBlockerModal = async () => {
+  await waitFor(() => {
+    expect(screen.getByTestId("modal-blocker-title")).toBeInTheDocument();
+  });
+};
+
+const expectElementContainsIcon = (id: string, ariaLabel: string) => {
+  const el = screen.getByTestId(id);
+  expect(el).toBeInTheDocument();
+  expect(el).toContainHTML(`aria-label="${ariaLabel}"`);
```

Comment:
beter: `expect(within(el).getByRole("img")).toHaveAccessibleName(ariaLabel);`

Reply:
would that still work if there are multiple images?

Reply:
Problem is that `IconError` does not have a `role=img` attribute. Will commit and push a fix. Curiously, Playwright doesn't seem to need an explicit `role` and presumable infers it from  the `svg` tag (and the accessible name?).

It's not important to test the exact html structure. It is important for the test to get as close as possible to what a person would see. So asserting that the aria label is on an element with role image is better than asserting thatit's on any element. What we indeed should not do, is asserting the `aside` element is present.